### ISSUE TITLE
Search box icon missing

### ISSUE THREAD

**Issue by dajoturner** [ucsc/wcms-design-templates#119]
The familiar magnifying glass icon is missing from the search box in Dev. It still says "Search..." but in my opinion it's less clear without the icon. If this is intentional, it should be added to the Design Change list.

Prod:
![image](https://cloud.githubusercontent.com/assets/8182013/3757675/ef592260-183b-11e4-8cff-11ad64bab349.png)

Dev:
![image](https://cloud.githubusercontent.com/assets/8182013/3757664/de12b804-183b-11e4-95fd-4ed970e3c104.png)
